cmake/mirrors/gitcode.cmake

```diff
@@ -19,11 +30,11 @@ ocv_update(ADE_PKG_MD5_ORIGINAL b624b995ec9c439cbc2e9e6ee940d3a2) # same as ade_
 # 1. Extract repo owner and repo name from DL_URL.
 # 2. Put repo owner and repo name into the placeholders of new DL_URL.
 #
-macro(ocv_download_url_gitcode_usercontent)
+macro(ocv_download_url_gitcode_usercontent OWNER)
   string(REPLACE "/" ";" DL_URL_split ${DL_URL})
   list(GET DL_URL_split 5 __COMMIT_ID)
   list(GET DL_URL_split 6 __PKG_NAME)
-  set(DL_URL "https://gitcode.net/opencv/opencv_3rdparty/-/raw/${__COMMIT_ID}/${__PKG_NAME}/")
+  set(DL_URL "https://gitcode.net/${OWNER}/opencv_3rdparty/-/raw/${__COMMIT_ID}/${__PKG_NAME}/")
```

Comment:
Both Gitcode and Github branches works for me well. But I see issues with double slashes in Gitcode links like this: `https://gitcode.net/mirrors/WeChatCV/opencv_3rdparty/-/raw/a8b69ccc738421293254aec5ddb38bd523503252//detect.caffemodel`. It does not happen in default (github) configuration.

Reply:
Fixed.